pybar_fei4_interpreter/Interpret.cpp

```diff
@@ -49,6 +49,7 @@ void Interpret::setStandardSettings()
 	_isMetaTableV2 = true;
 	_alignAtTriggerNumber = false;
 	_useTriggerTimeStamp = false;
+	_TriggerFormat = "COMBINED";

```

Comment:
I would prefer to have a number indicating the mode instead of a character array.